dm/dm/master/scheduler/scheduler.go

```diff
@@ -806,6 +806,20 @@ func (s *Scheduler) AddSubTasks(latched bool, cfgs ...config.SubTaskConfig) erro
 		m[stage.Source] = stage
 	}
 
+	// 6. check if we can resolve load task by scheduling the source to the worker which has dump files on local disk.
+	for _, cfg := range newCfgs {
+		source := cfg.SourceID
+		worker, ok := s.bounds[source]
+		if !ok {
+			continue
+		}
+
+		if err2 := s.tryResolveLoadTask(worker.BaseInfo().Name, source); err2 != nil {
+			s.logger.Error("in AddSubTasks, error when try resolve load task",
```

Comment:
why we can mute this error ?

Reply:
I think if it's not a big problem if it's failed, the replication can still working for most tasks.

rest comment https://github.com/pingcap/tiflow/pull/4004/commits/9e816129c8f7048a4f265ea999da0b08b72cec24, ptal